freqtrade/plugins/pairlist/VolumePairList.py

```diff
@@ -103,6 +141,54 @@ def filter_pairlist(self, pairlist: List[str], tickers: Dict) -> List[str]:
         # Use the incoming pairlist.
         filtered_tickers = [v for k, v in tickers.items() if k in pairlist]
 
+        # get lookback period in ms, for exchange ohlcv fetch
+        if self._use_range:
+            since_ms = int(arrow.utcnow()
+                           .floor('minute')
+                           .shift(minutes=-(self._lookback_period * self._tf_in_min)
+                                  - self._tf_in_min)
+                           .float_timestamp) * 1000
```

Comment:
why not use int_timestamp instead of explicitly parsing to int ... ?

Reply:
Honestly, I stole that part exactly from RangeStabilityFilter.py so, yeah sure this makes more sense, but maybe we should also look at the other filters, to improve on that? ;)

Reply:
for sure :laughing: